src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -123,6 +123,17 @@
       <RunTestsForProjectInputs Include="@(IntermediateAssembly)" />
       <RunTestsForProjectInputs Include="@(_DebugSymbolsIntermediatePath)" />
       <RunTestsForProjectInputs Include="@(AllItemsFullPathWithTargetPath)" />
+
+      <OverridenRuntimeContent Condition="'$(BUILDTOOLS_OVERRIDE_RUNTIME)' != ''" Include="$(BUILDTOOLS_OVERRIDE_RUNTIME)/*.*" />
+
+      <SourcesToCopyToTestDir Include="@(RunTestsForProjectInputs)" />
```

Comment:
I know see that Source and Destination are parallel item groups to later be passed to the Copy task. I think I would instead prefer we use item metadata for this instead of keeping two different item groups in-sync.